tests/test_matrix_operations.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+import numpy as np
+from polyssifier.polyssifier import exponent_matrix, create_multivariate
+
+@pytest.mark.medium
+def test_exponent_matrix():
+    matrix = np.matrix('1 2; 3 4')
+    matrix_2pow = np.matrix('1 4; 9 16')
+    matrix_3pow = np.matrix('1 8; 27 64')
+    matrix_4pow = np.matrix('1 16; 81 256')
+    assert (matrix == exponent_matrix(matrix, 1)).all()
+    assert (matrix_2pow == exponent_matrix(matrix, 2)).all()
+    assert (matrix_3pow == exponent_matrix(matrix, 3)).all()
+    assert (matrix_4pow == exponent_matrix(matrix, 4)).all()
+
+@pytest.mark.medium
+def test_create_multivariate():
+    matrix = np.matrix('5 6; 7 8')
```

Comment:
Please use numpy arrays instead of matrix. This way it will be consistent across all the code